contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -176,9 +178,16 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     if (fallbackWeiPerUnitLink <= 0) {
       revert InvalidLinkWeiPrice(fallbackWeiPerUnitLink);
     }
-    if (fulfillmentFlatFeeNativePPM > 0 && fulfillmentFlatFeeLinkDiscountPPM >= fulfillmentFlatFeeNativePPM) {
+    if (
+      fulfillmentFlatFeeLinkDiscountPPM > fulfillmentFlatFeeNativePPM ||
+      (fulfillmentFlatFeeNativePPM > 0 && fulfillmentFlatFeeLinkDiscountPPM == fulfillmentFlatFeeNativePPM)
+    ) {
       revert LinkDiscountTooHigh(fulfillmentFlatFeeLinkDiscountPPM, fulfillmentFlatFeeNativePPM);
     }
+    // After payment, there is at least one SLOAD and one SSTORE
+    if (gasAfterPaymentCalculation < 4000) {
+      revert InvalidGasAfterPaymentCalculation(gasAfterPaymentCalculation, 4000);
+    }
```

Comment:
So for `fulfillmentFlatFeeNativePPM` and `fulfillmentFlatFeeLinkDiscountPPM` - do we have some clear idea of the bounds expected for these parameters? Same question for `minimumRequestConfirmations`, `maxGasLimit` and `stalenessSeconds`.

Reply:
Yeah, we'll have to discuss that for this ticket. I started this PR to have ongoing discussions of good working ranges of these values:
minimumRequestConfirmations: [0, 200] // from comments of requestRandomWords
maxGasLimit: [5000, 1_000_000] // Minimum should be the gas cost of a contract call, max is debatable
stalenessSeconds: [1, 1_000_000] // Again, this range is debatable

Reply:
is it worth setting limits for these configs `minimumRequestConfirmations`, `maxGasLimit` and `stalenessSeconds`? 

IMO, it's generally hard to come up with reasonable value here without knowing what chains we are expanding to in the future. I think we shouldn't worry too much about these and just add validation for  `fulfillmentFlatFeeLinkDiscountPPM` and `fulfillmentFlatFeeNativePPM` because those 2 are actually related to each other and more prone to mistakes. Thoughts?

Reply:
@jinhoonbang Makes sense, it's very hard to determine "good enough" limits for these 3 parameters.

Reply:
Sure, nothing to update for the 3 params `minimumRequestConfirmations, maxGasLimit and stalenessSeconds`

How about `fulfillmentFlatFeeLinkDiscountPPM and fulfillmentFlatFeeNativePPM`? Do we need to add bounds for these variables

Reply:
Based on offline discussions, we would prefer not having any bounds for fulfillmentFlatFeeLinkDiscountPPM  fulfillmentFlatFeeNativePPM gasAfterPaymentCalculation`, as they are arbitrary